functions.php

```diff
@@ -598,10 +598,19 @@ function edit_searchwp_query_orderby( $orderby ) {
 	public function populate_act_page_children_options() {
 
 		// Get the id of the ACT page. We need this to get the children posts/pages of the ACT Page.
-		$arguments = [
-			'post_type'     => 'page',
-			'post_name__in' => [ 'act', 'ACT', 'Act' ],
-		];
+		$parent_act_id = planet4_get_option( 'select_act_page' );
```

Comment:
@sagarsdeshmukh Correct me if I am wrong but you do have the chosen Act Parent page id here and then you are doing things in lines 603-621 just to get that id that you already have?

Reply:
Yes agree, I optimize the populate_act_page_children_options() function.